fancy_prompt/fancy_prompt.py

```diff
@@ -35,8 +35,10 @@ def invoke(argument, from_tty):
         global colour_enabled
         if argument == 'on':
             colour_enabled = True
+            return
         elif argument == 'off':
             colour_enabled = False
+            return
```

Comment:
This is a small style thing but I think it's clearer in cases like this if it's:
```
if ...:
    foo()
elif ...:
    bar()
else:
    raise ValueError()
```

This is because the `raise` is an alternative to the others.

I prefer early returns when it's some condition checking for instance:
```
if something_is_not_ok:
    return:

if cannot_do_something:
    raise Foo()

... # normal code.
```

Reply:
Fixed